pkg/cluster/px/px.go

```diff
@@ -527,18 +527,33 @@ func (ops *pxClusterOps) upgradePX(spec apiv1beta1.ClusterSpec, opts *UpgradeOpt
 			{
 				APIGroups: []string{""},
 				Resources: []string{"persistentvolumeclaims", "persistentvolumes"},
-				Verbs:     []string{"get", "list"},
+				Verbs:     []string{"get", "list", "create", "delete", "update"},
 			},
 			{
 				APIGroups: []string{"storage.k8s.io"},
-				Resources: []string{"storageclasses"},
+				Resources: []string{"storageclasses", "csinodes"},
 				Verbs:     []string{"get", "list"},
 			},
+			{
+				APIGroups: []string{"storage.k8s.io"},
+				Resources: []string{"volumeattachments"},
+				Verbs:     []string{"get", "list", "create", "delete", "update"},
+			},
 			{
 				APIGroups: []string{""},
 				Resources: []string{"configmaps"},
 				Verbs:     []string{"get", "update", "list", "create"},
 			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"services"},
+				Verbs:     []string{"get", "list", "create", "update", "delete"},
```

Comment:
nit: keep the order of verbs consistent everywhere for easy comparison

nit: should we combine services and endpoints in the same block like we do pv and pvcs?